airbyte-webapp/src/packages/cloud/cloudRoutes.tsx

```diff
@@ -92,7 +95,7 @@ const MainRoutes: React.FC = () => {
         <Route path={`${RoutePaths.Settings}/*`} element={<CloudSettingsPage />} />
         <Route path={CloudRoutes.Credits} element={<CreditsPage />} />
 
-        {workspace.displaySetupWizard && (
+        {(workspace.displaySetupWizard || isExperimentVariant) && (
```

Comment:
We'll redirect users to the onboarding flow if they click on the banner CTA, therefore we need this route accessible